dapp/src/insurance/testflightdelay.sol

```diff
@@ -0,0 +1,745 @@
+/*
+
+	FlightDelay with Oraclized Underwriting and Payout
+	All times are UTC.
+	Copyright (C) Christoph Mussenbrock, Stephan Karpischek
+	
+*/
+
+
+import "./github.com/oraclize/ethereum-api/oraclizeAPI.sol";
+import "./github.com/Arachnid/solidity-stringutils/strings.sol";
```

Comment:
It is also highly recommended not to import code from URLs, those files could be modified by their owner without the end user noticing it